config/horizon.php

```diff
@@ -146,17 +146,17 @@
                 'connection' => 'redis',
                 'queue' => ['bpmn'],
                 'balance' => 'simple',
-                'processes' => 16,
-                'tries' => 1,
-                'timeout' => 3600,
+                'processes' => env('PM4_HORIZON_SUPERVISOR_BPMN_PROCESSES', 4),
+                'tries' => env('PM4_HORIZON_SUPERVISOR_BPMN_TRIES', 1),
```

Comment:
At this moment, this parameter `tries` must always be always `1`, since retry multiple times a BPMN Action could cause a token to be injected to the next task in every retry.